scripts/format-ros2-manual.sh

```
@@ -0,0 +1,39 @@
+echo ""
+echo "Option 2: Format Individual Packages Manually"
+echo "----------------------------------------------"
+echo "In the devcontainer terminal, run:"
+echo ""
+
+# Find all ROS2 packages
+if [ -d "star-ros2/src" ]; then
+    for pkg_xml in star-ros2/src/*/package.xml; do
+        if [ -f "$pkg_xml" ]; then
+            pkg_dir=$(dirname "$pkg_xml")
+            pkg_name=$(basename "$pkg_dir")
```

Shellcheck (0.11.0)

[warning] pkg_name appears unused. Verify use (or export if used externally).

(SC2034)